frontend/src/components/Home.tsx

```diff
@@ -0,0 +1,87 @@
+import { Avatar, Grid, Typography, List, ListItem, ListItemText, ListItemIcon, Container } from '@mui/material';
+import ContactPhoneIcon from '@mui/icons-material/ContactPhone';
+import AlternateEmailIcon from '@mui/icons-material/AlternateEmail';
+import DateRangeIcon from '@mui/icons-material/DateRange';
+import useFetch from 'use-http'
+
+interface User {
+    id: number;
+    username: string;
+    email: string;
+    image_url: string;
+    firstName: string;
+    lastName: string;
+    middleName: string;
+    phoneNumber: string;
+    gender: string;
+    created_at: string;
+}
+  
+const Home = () => {
+    const { loading, error, data: user } = useFetch('http://localhost:8000/auth/profile', {
```

Comment:
@ermyas7 what am i missing here? the user state (data) is not refreshed after the response ready. @abelbryo ping.